src/ir_Coolix.cpp

```diff
@@ -132,14 +132,16 @@ uint32_t IRCoolixAC::getRaw() { return remote_state; }
 
 void IRCoolixAC::setRaw(const uint32_t new_code) {
   if (!handleSpecialState(new_code)) {
-    // it isn`t special so might afect Temp|mode|Fan
+    // it isn`t special so might affect Temp|mode|Fan
     if (new_code == kCoolixCmdFan) {
       setMode(kCoolixFan);
     } else {
       // must be a command changing Temp|Mode|Fan
       // it is safe to just copy to remote var
       remote_state = new_code;
     }
+  } else {
+    remote_state = new_code;
   }
 }
 
```

Comment:
This would be slightly smaller, but obviously still requires the comments as it's not as obvious as most protocols.
```
void IRCoolixAC::setRaw(const uint32_t new_code) {
  if (!handleSpecialState(new_code)) {
    // it isn`t special so might affect Temp|mode|Fan
    if (new_code == kCoolixCmdFan) {
      setMode(kCoolixFan);
      return;
    }
  }
  // must be a command changing Temp|Mode|Fan
  // it is safe to just copy to remote var...
  remote_state = new_code;
}
```

Reply:
D'oh! Yes, you're right. Done.